Makefile

```diff
@@ -225,20 +232,18 @@ postgres: bin/waitfor
 		-p 5432:5432 \
 		docker.io/library/postgres:13-alpine && ./bin/waitfor "$(DB_URL)" && make regendb) || ($(CONTAINER_TOOL) start goalert-postgres && ./bin/waitfor "$(DB_URL)")
 
-regendb: bin/resetdb bin/goalert config.json.bak
+regendb: bin/resetdb bin/goalert config.json.bak ## Regenerate the database
 	./bin/resetdb -with-rand-data -admin-id=00000000-0000-0000-0000-000000000001 -mult $(SIZE)
 	test -f config.json.bak && bin/goalert set-config --allow-empty-data-encryption-key "--db-url=$(DB_URL)" <config.json.bak || true
 	bin/goalert add-user --user-id=00000000-0000-0000-0000-000000000001 --user admin --pass admin123 "--db-url=$(DB_URL)"
 
-resetdb: config.json.bak
+resetdb: config.json.bak ## Reset the database
 	go run ./devtools/resetdb --no-migrate
 
-clean:
+clean: ## Clean up the application
+	@go mod tidy
```

Comment:
This shouldn't introduce any changes, only clean up build artifacts (e.g., similar to a fresh `git clone`). This should be done in `check` instead

```suggestion
```